app/models/feed.rb

```diff
@@ -125,4 +142,29 @@ def self.notify_subscribers(pm, title, summary, url)
       end
     end
   end
+
+  private
+
+  def set_user_and_team
+    self.user ||= User.current
+    self.team ||= Team.current
+  end
+
+  def licenses_in_allowed_values
+    unless (self.licenses - LICENSES.keys).empty?
+      errors.add(:licenses, I18n.t(:"errors.messages.invalid_feed_licenses_value"))
+    end
+  end
+
+  def saved_search_belongs_to_feed_teams
+    unless saved_search_id.blank?
+      team_ids = self.feed_teams.map(&:team_id)
+      team_ids << self.team_id
```

Comment:
Suggestion: Implement a method `team_ids` in `Feed` that contains this logic... I saw this being used elsewhere.

Reply:
Done and add a method `get_team_ids`